common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -79,4 +82,9 @@ public static void sleepMs(Logger logger, long timeMs, boolean shouldInterrupt)
     Constructor<T> ctor = cls.getConstructor(ctorClassArgs);
     return ctor.newInstance(ctorArgs);
   }
+
+  public static int getToMasterHeartBeatIntervalMs(TachyonConf tachyonConf) {
+    return tachyonConf
+        .getInt(Constants.WORKER_TO_MASTER_HEARTBEAT_INTERVAL_MS, Constants.SECOND_MS);
+  }
```

Comment:
This doesn't seem too much of a utility method (aside from reducing the line length), perhaps add a TODO to remove this and just use the actual `getInt`